vertx-mongo-client/src/main/java/io/vertx/ext/mongo/impl/config/ConnectionPoolSettingsParser.java

```diff
@@ -12,7 +12,7 @@
 class ConnectionPoolSettingsParser {
 
   private final ConnectionPoolSettings settings;
-
+  private final Integer DEFAULT_MONGO_DRIVER_WAIT_Q_SIZE=100;
```

Comment:
OK, so maybe set this to 500 and reference the mongo driver variable in a comment.